data/mode.txt

```diff
@@ -1 +1 @@
-extended
+traitor
```

Comment:
Let's not.

Reply:
Though this directory was `.gitignore`'d, whoops